pymc4/coroutine_model.py

```diff
@@ -0,0 +1,236 @@
+import biwrap
+import functools
+import types
+import pymc4
+from pymc4.scopes import name_scope
+
+
+@biwrap.biwrap
```

Comment:
Since biwrap is so small, can we just add it to utils rather than having an external dependency?

Reply:
We can